docs/src/_data/further_reading_links.json

```
@@ -698,5 +698,19 @@
         "logo": "https://eslint.org/apple-touch-icon.png",
         "title": "Interesting bugs caught by no-constant-binary-expression - ESLint - Pluggable JavaScript Linter",
         "description": "A pluggable and configurable linter tool for identifying and reporting on patterns in JavaScript. Maintain your code quality with ease."
+    },
+    "https://tc39.es/ecma262/#sec-symbol-object": {
```

```suggestion
    "https://tc39.es/ecma262/#sec-symbol-constructor": {
```

This should fix the docs site build.